canonical-web-and-design/dotrun
# Make install scripts work multiple times

So that anyone can run the script regardless of if they already have dotrun installed, to try to fix any missing configuration (like linking docker to dotrun)

QA
--

Run it once:

```
scripts/install-dotrun-docker.sh
```

Run it again:

```
scripts/install-dotrun-docker.sh
```

Check it runs fine both times, and dotrun is installed and stuff.
